app/assets/stylesheets/application.css

```diff
@@ -1,15 +1,54 @@
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
- * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
- */
+body {
+  background-color: #333;
+  color: #fff;
+}
+
+.home {
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  width: 30%;
+}
+
+.user-container {
+  display: flex;
+  gap: 2rem;
+}
+
+.user-img {
+  border: 0.5px solid #131212;
+}
+
```

Comment:
- [ ] Fantastic progress so far! 💪✨ Please ensure the image styling aligns precisely with the wireframe guidelines Currently, when I'm using a valid image URL, this is how the image is being displayed 👇

![image](https://github.com/Innocent259/blog-app/assets/106119105/abf067aa-2992-4273-a66a-2bea04196663)